content_scripts/vimium_frontend.coffee

```diff
@@ -41,14 +41,10 @@ textInputXPath = (->
   DomUtils.makeXPath(inputElements)
 )()
 
-#
-# settings provides a browser-global localStorage-backed dict. get() and set() are synchronous, but load()
-# must be called beforehand to ensure get() will return up-to-date values.
-#
+# NOTE(mrmr1993): we use Settings everywhere instead of the dedicated implementation that was here.
+# Previously, only the values listed below would be loaded. If the space used by settings across all of our
+# content scripts is becoming an issue, then we can restrict the values we load to the list below.
 settings =
```

Comment:
It's probably safe enough to remove these (and the comment above).  If you want to keep them, then they could at least be commented out, could they not?